ai-platform-unified/notebooks/official/explainable_ai/gapic-custom_image_classification_online_explain.ipynb

```diff
@@ -31,7 +31,7 @@
     "repo": "snippets_housekeeping.ipynb"
```

Comment:
<p>google-cloud-storage if you mean the library</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>done</p>